chirp/drivers/ft1d.py

```diff
@@ -1131,7 +1131,8 @@ def _get_aprs_beacons(self):
                 checksum = body[-2:]
                 body = ''.join(s for s in body[:-2]
                                if s in string.printable).translate(
-                                   None, "\x09\x0a\x0b\x0c\x0d")
+                                   str.maketrans("", "",
+                                                 "\x09\x0a\x0b\x0c\x0d"))
```

Comment:
I don't know what this is trying to do... Can you explain please?

Reply:
As far as I can understand you are using translate() to remove these chars "\x09\x0a\x0b\x0c\x0d". But in python 3 translate needs a table. The recommended way to create the table is with maketrans(). maketrans could work with only one argument ("\x09\x0a\x0b\x0c\x0d") but I though it would be better to use the optional fields too. Am I missing something?

Reply:
Roger I see. Not sure this is the best way to remove characters we don't want from the string, but it's what was there before, so, okay.